src/metadata.rs

```diff
@@ -560,6 +573,14 @@ pub fn sort_year(i1: &Info, i2: &Info) -> Ordering {
     i1.year.cmp(&i2.year)
 }
 
+pub fn sort_filename(i1: &Info, i2: &Info) -> Ordering {
+    i1.file_name().cmp(&i2.file_name())
+}
+
+pub fn sort_filepath(i1: &Info, i2: &Info) -> Ordering {
+    i1.file_path().cmp(&i2.file_path())
```

Comment:
`PathBuf` implements `Ord`, so you could just write `i1.file.path.cmp(&i2.file.path)`.